spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/encryption/CipherEnvironmentEncryptor.java

```diff
@@ -64,8 +64,11 @@ private Environment decrypt(Environment environment, TextEncryptorLocator encryp
 			for (Map.Entry<Object, Object> entry : new LinkedHashSet<>(map.entrySet())) {
 				Object key = entry.getKey();
 				String name = key.toString();
-				String value = entry.getValue().toString();
-				if (value.startsWith("{cipher}")) {
+				String value = null;
+				if(entry.getValue() != null){
+				  value = entry.getValue().toString();		
+				}
+				if (value != null && value.startsWith("{cipher}")) {
```

Comment:
remove the `value` `null ` check and just keep this `if` wrapped in the above `if`